### ISSUE TITLE
How to Use include-path-file

### ISSUE THREAD

**Issue by ProfessorManhattan** [docker-slim/docker-slim#193]
Is there any documentation available on how to use `--include-path-file`? Do I just have a txt file with a list of paths to preserve right next to the Dockerfile?

**Comment by kcq** [comment]
@ProfessorManhattan yes, put a list of files or directories you want to keep. Make sure you specify absolute paths. For example:

```
/etc
/lib/lsb/init-functions.d/50-ubuntu-logging
```

In this example I'm including the `/etc` directory and the `/lib/lsb/init-functions.d/50-ubuntu-logging` file in my optimized image.

**Comment by ProfessorManhattan** [comment]
thanks @kcq -- I'm still unsure of what the difference between --include-path and --include-path-file is.

**Comment by kcq** [comment]
@ProfessorManhattan  the `--include-path-file` is a good option if you have a lot of includes and you don't want to have separate `--include-path` flags for each of them.

Here are two ways to accomplish the same results:

`docker-slim build --include-path /etc --include-path /lib/lsb/init-functions.d/50-ubuntu-logging TARGET_IMAGE`

`docker-slim build --include-path-file LOCAL-FILE-WITH-INCLUDES TARGET_IMAGE`

where `LOCAL-FILE-WITH-INCLUDES` is a local file on your host that has this data:
`
/etc
/lib/lsb/init-functions.d/50-ubuntu-logging
`
